ydb/docs/ru/core/yql/reference/yql-core/types/toc_i.yaml

```
@@ -3,6 +3,8 @@ items:
   href: index.md
 - name: Простые
   href: primitive.md
+- name: Serial
```

```suggestion
- name: Серийные
```
может тоже перевести, чтобы не выбивалось из списка?

тогда в остальных местах тоже консистентно надо будет сделать

---

Может все-таки перевести как автоинкрементные?

---

@shnikd afaik `serial` — скорее обобщение autoincrement, т.к. низлежащая последовательность не обязана именно инкрементить и может что-нибудь другое делать типа https://en.wikipedia.org/wiki/Snowflake_ID

но пока у нас альтернатив нет я не сильно против и автоинкремента, просто потом придётся переделывать вероятно